File: kombarika/com/dao/test/entity/District.cs
namespace com.dao.test.entity;


using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;



[Table(name = "district")]
public class District {

	[Column("nom_district")]
	string nomDistrict { get; set; }
	[ForeignKey("id_region")]
	Region region { get; set; }
	[Key]
	[Column("id")]
	int id { get; set; }




	public District(){}



}

File: kombarika/back/com/district/test/entity/District.java
package com.district.test.entity;

import com.dao.annotation.*;
import com.dao.annotation.conf.ForeignType;


@Table(name = "district")
public class District {
	@Column(name = "nom_district")
	String nomDistrict;

	@ForeignKey(mappedBy = "id_region", foreignType = ForeignType.OneToMany)
	Region idRegion;

	@PrimaryKey
	@GeneratedValue(autoIncrement = true)
	@Column(name = "id")
	Integer id;



	public District(){}

	public String getNomDistrict(){
		return this.nomDistrict;
	}
	public void setNomDistrict(String nomDistrict){
		this.nomDistrict = nomDistrict;
	}

	public Region getIdRegion(){
		return this.idRegion;
	}
	public void setIdRegion(Region idRegion){
		this.idRegion = idRegion;
	}

	public Integer getId(){
		return this.id;
	}
	public void setId(Integer id){
		this.id = id;
	}


}
